package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@stripe/react-connect-js",
-  "version": "3.3.9",
+  "version": "3.3.10",
```

Comment:
Why is this bumped in this PR? This should be bumped during publishing only

Reply:
This should have been bumped last time it seems. React-Connect-JS is currently published as 3.3.10.

Reply:
Let's bump it in a separate PR